common/types/json_list.go

```diff
@@ -20,6 +20,9 @@ import (
 	"github.com/google/cel-go/common/types/ref"
 	"github.com/google/cel-go/common/types/traits"
 	"reflect"
+	"github.com/golang/protobuf/ptypes"
```

Comment:
nit: imports should be alphabetically ordered.

Reply:
done.